WordPress/build.gradle

```diff
@@ -325,7 +328,7 @@ dependencies {
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
     implementation 'androidx.exifinterface:exifinterface:1.0.0'
     implementation 'androidx.media:media:1.0.1'
-    implementation 'androidx.appcompat:appcompat:1.1.0'
+    implementation 'androidx.appcompat:appcompat:1.6.0-rc01'
```

Comment:
Cc @ParaskP7 I know you had in your backlog to look into updating AppCompat at some point (in part exactly for this use case of supporting per-app locale backwards compatibility); seems @ravishanker beat you to it, which means (assuming no other changes due to this AppCompat update in other parts/topics in the app are needed) you might be able to cross an item off your list once this lands?